Bottom: 801d8630abba5450eb14d237677ee38b2bfbd05f
Top:    89d1f7bd64ee5689c203515eeb805bc0ebdff2db
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-23 20:56:37 +0000

Refresh of find-c-fix-inverted-logic-in-l

---

diff --git a/src/find.c b/src/find.c
index a1d6bce..3622777 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3067,7 +3067,7 @@ LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int which_flag,
   if (type == NO_TYPE)
     {
       type = SearchObjectByLocation (
-        LOOKUP_MORE & ~(AndRats ? RATLINE_TYPE : 0),
+        LOOKUP_MORE & ~(AndRats ? 0 : RATLINE_TYPE),
         &ptr1, &ptr2, &ptr3, X, Y, Range);
       if (type == NO_TYPE)
         return;
